recipes/dipy/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.11.0" %}
+
+package:
+  name: dipy
+  version: {{ version }}
+
+source:
+  fn: dipy-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/d/dipy/dipy-{{ version }}.tar.gz
+  md5: 7723e8994d31e910d47b7a0cf064a3d8
+
+build:
+    number: 0
+    script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - numpy >=1.7.1
```

Comment:
Actually, I check `setup.py` and I believe you need `numpy x.x` to ensure the numpy used in the build will be installed at run time.